### ISSUE TITLE
Since when / why does yt-dlp now output "[download]" messages in STDERR instead of STDOUT?

### ISSUE THREAD

**Issue by hasezoey** [yt-dlp/yt-dlp#3844]
### Checklist

- [X] I'm asking a question and **not** reporting a bug/feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones

### Question

It seems like yt-dlp at sometime in the last months started to output the `[download]` messages in STDERR instead of STDOUT, while having `--newline` and `--progress` set

Full Command:
```sh
yt-dlp --download-archive somefile.txt \
-f bestaudio/best -x --audio-format mp3 \
--embed-thumbnail --add-metadata --write-thumbnail \
--print "before_dl:PARSE_START '%(extractor)s' '%(id)s' %(title)s" --print "after_video:PARSE_END '%(extractor)s' '%(id)s'" \
--progress --newline --no-simulate \
-o "'%(extractor)s'-'%(id)s'-%(title)s.%(ext)s" \
SOMEYOUTUBEURL
```

### Verbose log

```shell
[debug] Command-line config: ['-vU']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2022.04.08 [7884ade65]
[debug] Python version 3.10.4 (CPython 64bit) - Linux-5.17.6-1-MANJARO-x86_64-with-glibc2.35
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0 (setts), ffprobe 5.0, rtmpdump 2.4
[debug] Optional libraries: certifi, Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
Latest version: 2022.05.18, Current version: 2022.04.08
ERROR: It looks like you installed yt-dlp with a package manager, pip or setup.py; Use that to update
```

**Comment by pukkandan** [comment]
All log except `--print` goes into stderr when you use it

**Comment by hasezoey** [comment]
> All log except `--print` goes into stderr when you use it. Without it, the progress is written to stdout

thanks, if there a way to force it to STDOUT or a `--print` formatting option for the download progress?

**Comment by pukkandan** [comment]
No

**Comment by UnixCro** [comment]
Don´t use  `-f b` or `-f best` anymore. It downloads Pre-merged formats, which are not the best!

**Comment by hasezoey** [comment]
> Don´t use `-f b` or `-f best` anymore. It downloads Pre-merged formats, which are not the best!

but it works as fallback, right?

**Comment by pukkandan** [comment]
It appears progress always went to `stdout` before `04.08`. So I may need to revert this. Re-opening this as a regression for now

**Comment by hasezoey** [comment]
TL;DR for those who come across: from what i could gather from the closing commit, the change that was made in 22.4.8 was reverted for the next version (probably being 22.6.8)

**Comment by gamer191** [comment]
> > Don´t use `-f b` or `-f best` anymore. It downloads Pre-merged formats, which are not the best!
> 
> but it works as fallback, right?

yes

**Comment by gamer191** [comment]
> TL;DR for those who come across: from what i could gather from the closing commit, the change that was made in 22.4.8 was reverted for the next version

correct

> (probably being 22.6.8)

yt-dlp currently doesn't have a set release schedule
